packages/vue/package.json

```diff
@@ -52,7 +52,9 @@
     "@babel/types": "^7.18.4",
     "@ionic/eslint-config": "^0.3.0",
     "@ionic/prettier-config": "^2.0.0",
+    "@rollup/plugin-node-resolve": "^16.0.0",
     "@rollup/plugin-typescript": "^11.1.5",
+    "@stencil/vue-output-target": "0.9.4",
```

Comment:
Does this need to be here when we have it here: https://github.com/ionic-team/ionic-framework/blob/0030be8195798fb097d931efd5c9913c99545780/core/package.json#L53

We don't do this for any of the other frameworks: https://github.com/ionic-team/ionic-framework/blob/0030be8195798fb097d931efd5c9913c99545780/packages/angular/package.json#L62

Reply:
> Does this need to be here when we have it here:

Yes, starting with `v0.9.0` we are shipping the Vue runtime as part of the package (see `import { defineContainer } from '@stencil/vue-output-target/runtime';` in [here](https://github.com/ionic-team/ionic-framework/blob/746b3380f8be87b6d01280217e0fc744b72df49e/packages/vue/src/proxies.ts#L3)). Since this code will be bundled with Rollup, we are ok keeping it as dev dependencies so for the Ionic user, nothing changes.

> We don't do this for any of the other frameworks:

We will have to do the same once we update React. Angular doesn't ship a runtime export.

Reply:
Thanks for the info! Sounds good.